datalake/src/main/resources/schema/app/20220304194213_CB-8676_Datalake_database_backup_stderr_output_has_non-error_messages.sql

```diff
@@ -0,0 +1,9 @@
+-- // CB-8676 Datalake database backup stderr output has non-error messages
+-- Migration SQL that makes the change goes here.
+
+alter table sdxoperation alter column statusreason type character varying(2000);
```

Comment:
Do you have a justification for this number? Just wondering since to an outsider it seems kind of random.